src/main/resources/migrations/00045_alter_table_gocardless_events.sql

```diff
@@ -0,0 +1,56 @@
+
+--changeset uk.gov.pay:alter_table_gocardless_events
```

Comment:
We usually have db migration scripts in their own PR, which will also help reduce the size of this one. By default the migration scripts do not run when deploying to staging and production and its necessary to manually invoke them. Would you mind splitting this out into it's own PR please?